nebari/template/stages/05-kubernetes-keycloak/modules/kubernetes/keycloak-helm/values.yaml

```
@@ -20,4138 +20,31 @@ startupScripts:
     /opt/jboss/keycloak/bin/add-user-keycloak.sh -r master -u nebari-bot -p "{{ .Values.nebari_bot_password }}"
 
 extraInitContainers: |
-    - command:
-      - sh
-      - -c
-      - mkdir -p /opt/jboss/keycloak/providers/ && cp /opt/keycloak/providers/keycloak-metrics-spi-2.5.3.jar
-          /opt/jboss/keycloak/providers/keycloak-metrics-spi-2.5.3.jar && chown 1000:1000
-          /opt/jboss/keycloak/providers/keycloak-metrics-spi-2.5.3.jar && chmod 777 /opt/jboss/keycloak/providers/keycloak-metrics-spi-2.5.3.jar
-      image: busybox:1.31
-      name: initialize-spi-metrics-jar
-      securityContext:
-        runAsUser: 0
-      volumeMounts:
-        - name: metrics-plugin
-          mountPath: /opt/keycloak/providers/
+  - command:
+    - sh
+    - -c
+    - |
+      wget https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar -P /data/ &&
+      export SHA256SUM=9b3f52f842a66dadf5ff3cc3a729b8e49042d32f84510a5d73d41a2e39f29a96 &&
+      if ! (echo "$SHA256SUM  /data/keycloak-metrics-spi-2.5.3.jar" | sha256sum -c)
+        then
+          echo "Error: Checksum not verified" && exit 1
+        else
+          chown 1000:1000 /data/keycloak-metrics-spi-2.5.3.jar &&
+          chmod 777 /data/keycloak-metrics-spi-2.5.3.jar
+      fi
```

Nit: formatting looks a little off

```suggestion
      if ! (echo "$SHA256SUM  /data/keycloak-metrics-spi-2.5.3.jar" | sha256sum -c); then
        echo "Error: Checksum not verified" && exit 1
      else
        chown 1000:1000 /data/keycloak-metrics-spi-2.5.3.jar &&
        chmod 777 /data/keycloak-metrics-spi-2.5.3.jar
      fi
```